docs/source/telegram.inline-tree.rst

```diff
@@ -1,6 +1,12 @@
 Inline Mode
 -----------
 
+The following methods and objects allow your bot to work in `inline mode`_.
```

Comment:
where does this link point to? Please make sure it points to https://core.telegram.org/bots/inline as it does in https://core.telegram.org/bots/api#inline-mode.